Source/WebCore/loader/ProgressTracker.cpp

```diff
@@ -121,21 +121,23 @@ void ProgressTracker::progressStarted(LocalFrame& frame)
         m_originatingProgressFrame = &frame;
 
         m_progressHeartbeatTimer.startRepeating(progressHeartbeatInterval);
-        RefPtr originatingProgressFrame = m_originatingProgressFrame;
-        originatingProgressFrame->protectedLoader()->loadProgressingStatusChanged();
+        if (RefPtr originatingProgressFrame = m_originatingProgressFrame.get()) {
```

Comment:
Oh, null check isn’t _actually_ necessary here, since we have `m_originatingProgressFrame = &frame;` above.